lms/lib/comment_client/user.py

```diff
@@ -5,6 +5,11 @@
 import models
 import utils
 
+import time
```

Comment:
Please rearrange imports according to the OpenEdx code style guide.
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order